packages/inventory/src/redux/entities.js

```diff
@@ -50,18 +51,21 @@ const defaultColumns = [
             value,
             _id,
             {
-                culled_timestamp: culled, stale_warning_timestamp: staleWarn, stale_timestamp: stale
+                culled_timestamp: culled, stale_warning_timestamp: staleWarn, stale_timestamp: stale, ...rest
             }) => {
             return CullingInformation ? <CullingInformation
                 culled={culled}
                 staleWarning={staleWarn}
                 stale={stale}
-                render={({ msg }) => <DateFormat date={ value } extraTitle={ (
-                    <React.Fragment>
-                        <div>{ msg }</div>
+                render={({ msg }) => <React.Fragment>
+                    <DateFormat date={ value } extraTitle={ (
+                        <React.Fragment>
+                            <div>{ msg }</div>
                         Last seen:{` `}
-                    </React.Fragment>
-                ) }/>
+                        </React.Fragment>
+                    ) }/>
+                    {rest.insights_id && <InsightsDisconnected />}
```

Comment:
Shouldn't this be the other way around? If `insights_id` is not present show the `InsightsDisconnected` component?

Reply:
It's a draft. :smile_cat: